Sources/MapboxCoreNavigation/NavSDK.swift

```diff
@@ -0,0 +1,41 @@
+import Foundation
+import MapboxDirections
+
+public struct NavSDK {
+    private static let lock: NSLock = .init()
+    private static var isInitialized: Bool = false
+    public static var directions: Directions!
+    public static var tileStoreConfiguration: TileStoreConfiguration!
```

Comment:
What if we use default values here and don't require calling `initialize` at the very start of the app?

Reply:
Yep, will do, thanks for suggestion. It will be the best of two worlds.

Reply:
Done, please review the latest revision.